mappings/net/minecraft/structure/OceanMonumentGenerator.mapping

```diff
@@ -15,16 +21,59 @@ CLASS net/minecraft/class_3366 net/minecraft/structure/OceanMonumentGenerator
 		METHOD method_14769 canGenerate (Lnet/minecraft/class_3366$class_3388;)Z
 			ARG 1 setting
 	CLASS class_3376 CoreRoom
+		METHOD <init> (Lnet/minecraft/class_2350;Lnet/minecraft/class_3366$class_3388;)V
+			ARG 1 orientation
+			ARG 2 setting
+		METHOD <init> (Lnet/minecraft/class_3485;Lnet/minecraft/class_2487;)V
+			ARG 1 structureManager
+			ARG 2 nbt
 	CLASS class_3377 DoubleXRoom
+		METHOD <init> (Lnet/minecraft/class_2350;Lnet/minecraft/class_3366$class_3388;)V
+			ARG 1 orientation
+			ARG 2 setting
+		METHOD <init> (Lnet/minecraft/class_3485;Lnet/minecraft/class_2487;)V
+			ARG 1 structureManager
+			ARG 2 nbt
 	CLASS class_3378 DoubleXYRoom
+		METHOD <init> (Lnet/minecraft/class_2350;Lnet/minecraft/class_3366$class_3388;)V
+			ARG 1 orientation
+			ARG 2 setting
+		METHOD <init> (Lnet/minecraft/class_3485;Lnet/minecraft/class_2487;)V
+			ARG 1 structureManager
+			ARG 2 nbt
 	CLASS class_3379 DoubleYRoom
 	CLASS class_3380 DoubleYZRoom
+		METHOD <init> (Lnet/minecraft/class_2350;Lnet/minecraft/class_3366$class_3388;)V
+			ARG 1 orientation
+			ARG 2 setting
+		METHOD <init> (Lnet/minecraft/class_3485;Lnet/minecraft/class_2487;)V
+			ARG 1 structureManager
+			ARG 2 nbt
 	CLASS class_3381 DoubleZRoom
+		METHOD <init> (Lnet/minecraft/class_2350;Lnet/minecraft/class_3366$class_3388;)V
+			ARG 1 orientation
+			ARG 2 setting
+		METHOD <init> (Lnet/minecraft/class_3485;Lnet/minecraft/class_2487;)V
+			ARG 1 structureManager
+			ARG 2 nbt
 	CLASS class_3382 Entry
+		METHOD <init> (Lnet/minecraft/class_2350;Lnet/minecraft/class_3366$class_3388;)V
+			ARG 1 orientation
+			ARG 2 setting
+		METHOD <init> (Lnet/minecraft/class_3485;Lnet/minecraft/class_2487;)V
+			ARG 1 structureManager
+			ARG 2 nbt
 	CLASS class_3383 Penthouse
+		METHOD <init> (Lnet/minecraft/class_2350;Lnet/minecraft/class_3341;)V
+			ARG 1 orientation
+			ARG 2 box
+		METHOD <init> (Lnet/minecraft/class_3485;Lnet/minecraft/class_2487;)V
+			ARG 1 structureManager
+			ARG 2 nbt
 	CLASS class_3384 Piece
 		FIELD field_14468 TWO_TWO_ZERO_INDEX I
 		FIELD field_14469 TWO_ZERO_ZERO_INDEX I
+		FIELD field_14470 ALSO_PRISMARINE_BRICKS Lnet/minecraft/class_2680;
```

Comment:
(not meant to be a blocking review comment) These could maybe get better names at some point in the future. Mojang has likely meant these two prismarine brick fields for two different use cases.

Reply:
Maybe usages like some is for floor and some is for roof? Things like that I guess?